javaee/app-profile-saml-jee-jsp/config/client-import.json

```diff
@@ -1,4 +1,4 @@
-{
+
```

Comment:
This looks like a typo? Parenthesis should not be removed?

Reply:
Done